src/content/roadmap/_index.en.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Roadmap"
+date = 2020-02-19T20:24:34+01:00
+weight = 100
+pre = "<b>5. </b>"
++++
+
+This is a preliminary community roadmap, it's not written in stone, but it can serve
+as a guideline about what's ahead.
+
+Please see details of previous releases [here](../releases)
+
+### v0.1.1
+
+* Stabilization of new lighthouse related features and deployment
+* Bugfixes
+* Working with the OpenShift CoreDNS distribution to make lighthouse compiled in their container
+
+### v0.2.0
+
+* Support for Overlapping CIDRs
+* Improve documentation and website
+* (internal) start using Armada to deploy multiple clusters in CI
+* Support for multiple cable engines, including wireguard
+
+### v0.3.0
+
+* Removing the kubefed dependency from Lighthouse service discovery
+* Measuring and improving A/P HA (different scenarios)
+* Libreswan support
+
+### v0.4.0 
+
+* Support for network policies via coastguard
+* Monitoring and reporting of tunnel endpoints
```

Comment:
Is this regarding validating if the corresponding ports (i.e., udp 4800) are opened between the worker nodes and worker nodes are able to talk to Gatewaynode over vxlan tunnel?

Reply:
No, more about monitoring the tunnel status with any information we can provide, and probably reporting to prometheus.

The other was more about fault/issue detection, I think it's worth we add something about that to the roadmap, thanks for the point, I will clarify this one and add your suggestion.